app.go

```diff
@@ -165,11 +165,59 @@ func (a *App) deleteProduct(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, http.StatusOK, map[string]string{"result": "success"})
 }
 
+func (a *App) getStoreProducts(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	id, err := strconv.Atoi(vars["id"])
+	if err != nil {
+		respondWithError(w, http.StatusBadRequest, "Invalid Store ID")
+		return
+	}
+
+	s := []storeProduct{}
+	s, err = getStoreProducts(a.DB, id)
+	if err != nil {
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	respondWithJSON(w, http.StatusOK, s)
+}
 
 func (a *App) initializeRoutes() {
 	a.Router.HandleFunc("/products", a.getProducts).Methods("GET")
 	a.Router.HandleFunc("/product", a.createProduct).Methods("POST")
 	a.Router.HandleFunc("/product/{id:[0-9]+}", a.getProduct).Methods("GET")
 	a.Router.HandleFunc("/product/{id:[0-9]+}", a.updateProduct).Methods("PUT")
 	a.Router.HandleFunc("/product/{id:[0-9]+}", a.deleteProduct).Methods("DELETE")
-}
\ No newline at end of file
+	a.Router.HandleFunc("/store/{id:[0-9]+}/products", a.getStoreProducts).Methods("GET")
+	a.Router.HandleFunc("/store/{id:[0-9]+}", a.createStore).Methods("POST")
+}
+
+func (a *App) createStore(w http.ResponseWriter, r *http.Request) {
```

Comment:
as products are being added to store here, can you consider changing the method name to **addProductsToStore**.

Reply:
Renamed the function to addProductToStore - Ref: [Commit 228b0e](https://github.com/nitinks-dnz/go-mux/commit/228b0eb1d2fb8c1bdbebaaedc983c36e7d04ca0d)